config/product-mapping.js

```diff
@@ -60,6 +87,15 @@ const productConfig = {
       modexProducts: [ { productName: "Expressen", productCode: "EX" } ],
       diCustomerSystem: "EXPA",
       diTitle: "EX Expressen",
+      deliveryDays: [
+        "monday",
+        "tuesday",
+        "wednesday",
+        "thursday",
+        "friday",
+        "saturday",
+        "sunday",
```

Comment:
A little bit out of scope, but when you're done with all this for Local/Gota, we'll be able to change Expressen's deliveryDays to remove `sunday` so that it matches the current reality 🎉